test/fgpu/test_delay.py

```diff
@@ -184,3 +200,31 @@ def test_range_empty(request, model) -> None:
     assert time.shape == (0,)
     assert residual.shape == (0,)
     assert phase.shape == (0,)
+
+
+def test_aligned_range(linear: LinearDelayModel, aligned: AlignedDelayModel, aligned_odd: AlignedDelayModel) -> None:
+    """Test :meth:`.AlignedDelayModel.range`."""
+    time1, residual1, phase1 = linear.range(13000, 14000, 3)
+    time2, residual2, phase2 = aligned.range(13000, 14000, 3)
+    time3, residual3, phase3 = aligned_odd.range(13000, 14000, 3)
```

Comment:
I see `phase3` isn't actually used, I gather you've 'stored it' just for completeness/feng shui?

Reply:
Good catch! I've added a couple of lines to fully check aligned_odd.